### ISSUE TITLE
Hosting static assets : using custom bucket

### ISSUE THREAD

**Issue by romainquellec** [danielcondemarin/serverless-nextjs-plugin#31]
I'm using this : 

```
plugins:
  - serverless-nextjs-plugin

custom:
  serverless-nextjs:
    nextConfigDir: ./
    assetsBucketName: assets.cuistotducoin.com
```

with (in next.config.js)
```
  assetPrefix: 'https://assets.cuistotducoin.com',
```
Looking at the bucket on AWS, its empty ! I deleted the stack and let serverless re-create it, but no change.

You can look the repo here : https://github.com/CuistotduCoin/front

Console SLS : 
```
Serverless Nextjs: assets bucket secure URL: https://assets.cuistotducoin.com.s3.amazonaws.com
Serverless Nextjs: assets bucket website URL: http://assets.cuistotducoin.com.s3-website-eu-west-1.amazonaws.com
```

**Comment by romainquellec** [comment]
FYI, the bucket is created, but empty. I have no idea why, and how this is different from the assetBucket (looking at the code).

**Comment by sja** [comment]
Same here, using 1.5.2

**Comment by danielcondemarin** [comment]
Hi @romainquellec @sja Sorry for not picking this up earlier, just published v1.5.3 with a fix for this!
